### DISCUSSION TITLE
Welcome to Hyprdots-ctl Discussions!

### DISCUSSION THREAD

**Discussion by kRHYME7** [kRHYME7/Hyde-cli#3] [Announcements]
## Hello everyone! 👋

Welcome to the Hyprdots-ctl repo!

You can post your questions and Feature requests in this section.
Your contributions are greatly appreciated! 🙏

**Comment by bluehysteric** [comment]
Unable to restore the backup, select the backup in the backup list to restore displays Operation Cancelled。
Unable to delete backups, selecting a backup in the backup list for deletion shows that it has been deleted, but the backup still exists.
